docs/source/tags/pdf-beta.md

```diff
@@ -0,0 +1,81 @@
+---
+title: PDF - Beta 🧪
+type: tags
+order: 307
+meta_title: PDF tag for loading PDF documents
+meta_description: Label Studio PDF tag for loading PDF documents for machine learning and data science projects.
+---
+
+!!! note
+    We’re testing a new PDF tag. To enable it for your account please contact your Customer Success Manager or apply here: [https://humansignal.com/pdf-interest-signup](https://humansignal.com/pdf-interest-signup)
+
+The `Pdf` tag displays a PDF document in the labeling interface. You can use this tag to:
+
+- Perform document-level annotations such as classification, transcription, and summarization. See [PDF](pdf#Example) for an example labeling config. 
+- Create regions within pages for OCR, NER, and other types of annotations (beta). <span class="badge"></span>
+
+Supports:
+* Zoom
+* Rotation
+* PDFs up to 100 pages
+
+Use with the following data types: PDF.
+
+!!! error Enterprise
+    You can also use the PDF tag with [Prompts](https://docs.humansignal.com/guide/prompts_overview) to perform auto-labeling work such as PDF summarization, classification, information extraction, and document intelligence. 
+
+    Note that since this tag is still in beta, Label Studio’s AI tools will default to the basic implementation of the tag that only supports document-level classification.
+
+{% insertmd includes/tags/pdf.md %}
+
+
+## Example: OCR 🧪 <span class="badge"></span>
+
+Beta and Label Studio Enterprise only.
+
+Labeling configuration to label PDF documents:
+
+```xml
+<View>
+  <Style>
+    .pdf-actions{display:flex;justify-content:space-between}
+  </Style>
+  <Header value="Label any areas that you wish to correct" size="4"/>
+  <View className="pdf-actions">
+    <OcrLabels name="ocr" toName="pdf">
+      <Label value="Typo"/>
+      <Label value="Incorrect amount"/>
+      <Label value="Incorrect name"/>
+    </OcrLabels>
+  </View>
```

Comment:
The idea here was to have labels on the left and per-region choices to the right. If we don't have them then better to remove the `View` and connected styles to make it simple.
But I think it would be better to add per-regions back to showcase possible use case.